frontend/src/components/pages/admin/DisplayAssessmentsPage.tsx

```diff
@@ -158,7 +158,7 @@ const DisplayAssessmentsPage = (): React.ReactElement => {
 
   const AssessmentTabPanels = [...Array(4)].map((i) => {
     return (
-      <TabPanel key={i}>
+      <TabPanel key={i} padding="0">
```

Comment:
we could also maybe define this styling in our `customTheme` for future tables (as well as the modal border radius)

but if its too complicated we can keep as is

Reply:
i tried to define it in themes before, but for some reason when i define padding here, it applies the padding to the text label, not the entire tab panel

Reply:
i think when i apply styling to `TabPanel` for some reason, it keeps applying to the text / tab itself and not the content of the tab if that makes sense